deploy/kubeturbo-operator/Makefile

```diff
@@ -143,6 +152,107 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
 
+
+YQ ?= $(LOCALBIN)/yq
+YQ_TOOLS_VERSION ?= v4.30.4
+
+.PHONY: yq
+yq: $(YQ) ## Download yq locally if necessary.
+$(YQ): $(LOCALBIN)
+	test -s $(LOCALBIN)/yq || GOBIN=$(LOCALBIN) go install github.com/mikefarah/yq/v4@$(YQ_TOOLS_VERSION)
+
+
+.PHONY: python
+PYTHON = $(LOCALBIN)/python3
+python: $(PYTHON)  ## Install Python locally if necessary. Darwin OS is specific to mac users if running locally
+$(PYTHON):
+	@if ! command -v python3 >/dev/null 2>&1; then \
+		mkdir -p $(LOCALBIN); \
+		if [ `uname -s` = "Darwin" ]; then \
+			brew install python@3; \
+		else \
+			sudo apt update && sudo apt install python3; \
+		fi \
+	fi
+	ln -sf `command -v python3` $(PYTHON)
+
+
+
+
+OPERATOR_VERSION_BETA ?= $(if $(OPERATOR_BETA_RELEASE_MINOR_VERSION),beta.$(OPERATOR_BETA_RELEASE_MINOR_VERSION),beta.1)
+OPERATOR_OLM_INCLUSIVE_RANGE_VERSION ?= 8.7.5
+OPERATOR_OLM_INCLUSIVE_BETA_VERSION ?= beta.1
+OPERATOR_RELEASE_FILTER ?= beta
+OPERATOR_CERTIFIED ?= kubeturbo-certified
+OPERATOR_BUNDLE_DIR ?= certified-operator-bundle
+OPERATOR_BUNDLE_TEMPLATE_DIR ?= bundle-template
+OPERATOR_CRD_FILE_PATH ?= deploy/crds/charts_v1alpha1_kubeturbo_crd.yaml
+CLUSTER_PERMISSION_ROLE_YAML_FILE_PATH ?= deploy/kubeturbo-operator-cluster-role.yaml
+CERTIFIED_OPERATOR_CLUSTER_SERVICE_VERSION_YAML_FILE_PATH ?= $(OPERATOR_BUNDLE_DIR)/manifest/kubeturbo-certified.clusterserviceversion.yaml
+
+.PHONY: build-certified-operator-bundle
+build-certified-operator-bundle:yq python verify-operator-release-channel verify_image_digest_version create_certified_operator_bundle_directory update_image_digest_in_operator_bundle update_operator_version_in_operator_bundle update_olm_skipRange_in_operator_bundle update_cluster_permissions_in_operator_bundle update_release_channel_in_operator_bundle
+## verify operator release channel, to allow only valid releases
+verify-operator-release-channel:
```

Comment:
seems all these targets are also `.PHONY`

Reply:
Maintaining own script file for each target may be confusing and hard to navigate between multiple files.  Thank you for your suggestion on making the targets as .PHONY, will check on that